ci/cloudbuild/build.sh

```diff
@@ -191,6 +191,13 @@ export TRIGGER_TYPE
 export VERBOSE_FLAG
 export LOG_LINKER_PAT
 
+# Default to generating all libraries.
+if [ -n "${GENERATE_GOLDEN_ONLY+set}" ]; then
+  GENERATE_GOLDEN_ONLY=${GENERATE_GOLDEN_ONLY:-}
+else
+  GENERATE_GOLDEN_ONLY=""
+fi
```

Comment:
This seems to be the equivalent of `: ${GENERATE_GOLDEN_ONLY=}`, which can be read as "if GENERATE_GOLDEN_ONLY is unset, then set it to the empty string".

But, you could just use `${GENERATE_GOLDEN_ONLY-}` below instead.